inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java

```diff
@@ -322,6 +328,34 @@ private void updateTaskStatus(CommandEntity command) {
         }
     }
 
+    @Override
+    public AgentConfigInfo getAgentConfig(AgentConfigRequest request) {
+        LOGGER.info("begin to get agent config info for {}", request);
```

Comment:
Will there be too many logs printed like this?

Reply:
Fixed.